statistics/update.go

```diff
@@ -92,6 +94,16 @@ func (s *SessionStatsCollector) StoreQueryFeedback(feedback interface{}) {
 	if q.histVersion == 0 || q.actual < 0 || !q.valid {
 		return
 	}
+
+	var rate float64
+	if q.actual == 0 {
+		rate = 100
+	} else {
+		rate = math.Abs(float64(q.expected-q.actual)/float64(q.actual)) * 100
+		rate = math.Min(100, rate)
```

Comment:
So when expected is way larger than actual, the rate is 100.

Reply:
Yes.